app/models/tagging.rb

```diff
@@ -43,4 +44,11 @@ def self.find_by_tag(taggable, tag)
   def update_taggings_count
     tagger.update_tag_cache unless tagger.blank? || tagger.destroyed?
   end
+
+  def update_search
+    reindex_boundary = 100
```

Comment:
This part has me a little confused. Will things just not get reindexed if there are more than 100 taggings? Could that cause any sort of issue?

Reply:
That makes sense to me, but I think we might want to run it by Tag Wrangling, since they were able to see all suggested fandoms on unwrangled tags before. If they're okay with the limitation, we can add a code comment that is basically what you said here.